GUI2.py

```diff
@@ -20,7 +20,7 @@
 which_potion = 0  # variable that lets you select the potion that you want to take.
 
 def get_room():
-    room = ("monster", "shop", "treasure box", "monster", "shop")
+    room = ("monster", "shop", "treasure box", "bossmonster", "shop")
```

Comment:
This is not feasible since we want a certain number of rooms to be crossed before entry is allowed in the boss room